### PR TITLE
codeclimate changes

### PR DESCRIPTION
**Pull request by kashyapgajera**

### CODE REVIEW

File: execution_nodes/execute_node.js [hunk 1]
```diff
@@ -1,80 +1,78 @@
...
-  console.log('connectionCheck requested');
+
+app.get("/connectionCheck", function (req,res) {
+  console.log("connectionCheck requested");
   res.send(true);
 });
 
-app.post('/requestRun', function(req, res){
-  console.log('requestRun post request recieved');
+app.post("/requestRun", function(req, res){
```
Comment (yash10p) [inline execution_nodes/execute_node.js]: There is no closing bracket '{' for this. This needs to be fixed.

File: execution_nodes/execute_node.js [hunk 2]
```diff
@@ -83,64 +81,44 @@ app.post('/requestRun', function(req, res){
...
+    body.submission_details.comment = comment;
+    body.submission_details.Lab_No = req.body.Lab_No;
+    body.submission_details.time = req.body.time;
+    body.submission_details.status = req.body.status;
+    body.submission_details.penalty = req.body.penalty;
+    body.submission_details.socket = req.body.socket;
+    body.submission_details.log = log_b64;
+    body = JSON.stringify(body);
+
+    request = https.request(https_request_options,function(res)
```
Comment (yash10p) [inline execution_nodes/execute_node.js]: Space needed after comma. There are similar instances where space needs to be added. Such as in between the multiple arguments given to a function. You can figure them out. http://eslint.org/docs/rules/comma-spacing

**Comment by prasadtalasila** [comment]
The suggestions made by @yash10p are valid. Also the code climate eslint is having a problem parsing the execute_node.js file. This error may be due to the missing set bracket and more?

Please run the linted code in your container, make sure that the execution node is functioning properly and the evaluation is happening properly.

Please make the suggested corrections.

**Comment by prasadtalasila** [comment]
Since @kashyapgajera deleted the branch and created a new fork, I am closing this pull request.

A fresh pull request from the new fork  is better.

**Pull request closed without merging**